api/src/graphql/oidc.sdl.js

```diff
@@ -9,7 +9,15 @@ export const schema = gql`
 
   type Mutation {
     createClient: Oidc! @requireAuth
-    updateClient(id: String!, redirectUrls: String!): Oidc! @requireAuth
+    updateClient(
+      id: String!
+      redirectUrls: String!
+      clientURI: String!
+      clientName: String!
+      tosURI: String!
+      logoURI: String!
+      policyURI: String!
```

Comment:
I don't think these should always be required, in case the developer doesn't have a link yet